components/inputs/input.vue

```diff
@@ -24,8 +24,8 @@
       <div
         class="relative"
         :class="[
-          wrapperClasses,
-          { 'mt-2': !dense, 'input-wrapper': !lg && !xl && !transparentBg }
+          { 'mt-2': !dense, 'input-wrapper': !lg && !xl && !transparentBg },
```

Comment:
This gets too messy over time, let's move it to a computed property, alongside with the `wrapperClasses`